spec/controllers/reports_controller_spec.rb

```diff
@@ -87,6 +87,61 @@
       end
 
     end
+
+    context "when reporting a comment (logged in)" do
+      before do
+        session[:user_id] = user.id
+      end
+
+      let(:comment) do
+        FactoryGirl.create(:comment, :info_request => info_request)
+      end
+
+      it "should mark the comment as having been reported" do
+        expect(comment.attention_requested).to eq(false)
+
+        post :create, :request_id => info_request.url_title,
+                      :comment_id => comment.id,
+                      :reason => "my reason"
+        expect(response)
+          .to redirect_to show_request_path(:url_title =>
+                                              info_request.url_title)
+
+        comment.reload
+        expect(comment.attention_requested).to eq(true)
+      end
+
+      it "should not mark the parent request as having been reported" do
+        expect(info_request.attention_requested).to eq(false)
+
+        post :create, :request_id => info_request.url_title,
+                      :comment_id => comment.id,
+                      :reason => "my reason"
+        expect(response)
```

Comment:
You've already got an assert for the redirect in the spec above.

Personally, I'd have a spec for each:

```ruby
it "marks the comment as having been reported" do
it "does not mark the parent request as having been reported" do
it "sends an email alerting admins to the report" do
it "redirects to the parent request" do
it "informs the user that the comment has been reported" do
```